### ISSUE TITLE
syntax highlighting is not really working

### ISSUE THREAD

**Issue by kbenoit** [quanteda/blog.quanteda.org#19]
The syntax highlighting for our R code does not appear to really be working. See for instance the presidential debate blog post. 

I say "not really" because there is a blue colour for the quoted strings, but that's it.

**Comment by stefan-mueller** [comment]
I agree that syntax highlighting in code chunks still needs improvement. [Here](https://gist.github.com/yihui/3422133) I found [this](http://animation.r-forge.r-project.org/knitr/) overview of available knitr syntax highlighting styles. Is it possible to use one of them for our blog posts?
